morituri/program/flac.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import check_call, CalledProcessError
+
+import logging
+logger = logging.getLogger(__name__)
+
+FLAC = 'flac'
+
+
+def encode(infile, outfile):
+    """
+    Encodes infile to outfile, with flac.
+    Uses '-f' because morituri already creates the file.
+    """
+    try:
+        check_call(['flac', '--totally-silent', '--verify', '-o', outfile,
+                    '-f', infile])
```

Comment:
I think you just proved that this is in fact bloat. We can easily ship a script that does it for users, but I think it's a bit silly to do this in our internal ripping routine. (Also, my bash script does lose the metadata, but you get the general idea)

I don't mind either way, but I would really prefer to first get this merged, in the most simple form possible, and then work on making it more complex/bloated/featured.

Discussion about these options and features belong in a issue report after this code is merged, IMHO.

Loving the feedback, though!